ci/gpu/build.sh

```diff
@@ -39,6 +39,21 @@ $CC --version
 $CXX --version
 conda list
 
+logger "Check GPU running the tests..."
+GPU="$(nvidia-smi | awk '{print $4}' | sed '8!d')"
+echo "Running tests on $GPU"
+
+if [[ $GPU == *"P100"* ]]; then
+  logger "Building for Pascal..."
+  GPU_ARCH="-DGPU_ARCHS=\"60\""
+elif [[ $GPU == *"V100"* ]]; then
+  logger "Building for Volta..."
+  GPU_ARCH=GPU_ARCH="-DGPU_ARCHS=\"70\""
+elif [[ $GPU == *"T4"* ]]; then
+  logger "Building for Turing..."
+  GPU_ARCH=GPU_ARCH="-DGPU_ARCHS=\"75\""
+fi
```

Comment:
Add else clause to build all

Reply:
if `GPU_ARCH` is not specified then the command builds for all architectures, therefore an else clause with all the architectures is not needed